scripts/build_binaries.sh

```diff
@@ -11,12 +11,46 @@ cd $script_dir/..
 docker run --rm -v `pwd`:/src golang:"$GO_BUILD_VERSION"-bookworm /bin/bash -c '
 set -e
 set -o pipefail
-apt-get update && apt-get install -y p7zip-full pigz
+apt-get update && apt-get install -y p7zip-full pigz curl xz-utils mingw-w64 clang-15
+
+cd /
+curl -o optcross.tar.xz https://dolthub-tools.s3.us-west-2.amazonaws.com/optcross/"$(uname -m)"-linux_20240425_0.0.1.tar.xz
+tar Jxf optcross.tar.xz
```

Comment:
tar -Jxf?

Reply:
Maybe! Works as is and I usually leave them off for `tar`...

Reply:
I'm going to leave it for now, since it matches usage elsewhere like in the Dolt repo.